documentation/modules/cruise-control/con-rebalance-performance.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='con-rebalance-{context}']
+
+= Rebalance performance tuning overview 
```

Comment:
I wonder if this should be in the APi reference (added usin the `@DescriptionFile` annotation). WDYT @PaulRMellor ?